src/main/resources/plugin.yml

```diff
@@ -5,3 +5,19 @@ api-version: 1.13
 authors: [ Zach_FR ]
 softdepend: [ ShopGUIPlus, HolographicDisplays, zShop, Vault, CMI, QuantumShop, BlockLocker, EconomyShopGUI, WorldGuard, WorldEdit, EpicShop, BossShopPro, EconomyShopGUI-Premium ]
 description: Sell wand plugin
+commands:
```

Comment:
ZachCore registers commands, this is unnecessary.

Reply:
The reason i added it is because LuckPerms doesn't tab-complete the perms without it